src/workerd/api/worker-loader.c++

```diff
@@ -170,11 +180,11 @@ Worker::Script::Source WorkerLoader::extractSource(jsg::Lock& js, WorkerCode& co
     KJ_UNREACHABLE;
   };
 
+  bool isPython = code.mainModule.endsWith(".py"_kj);
```

Comment:
We have other "isPython" checks which return true if the bundle contains _any_ Python modules, but here you're only checking the main module. I imagine in practice a Python worker must have a Python main module, but I worry the inconsistency could lead to bugs if people specify a worker containing a weird mix of modules.

* What happens if your main module is JS, but you have Python modules?
* What happens if your main module is Python, but you have JS modules?

Do either of these two configurations make sense today? If not, can we explicitly enforce that you not do that? That both avoids bugs and frees our hand to decide to support one of these configurations in a more thoughtful way in the future.